tfe/resource_tfe_policy_set.go

```diff
@@ -46,6 +46,24 @@ func resourceTFEPolicySet() *schema.Resource {
 				ConflictsWith: []string{"workspace_ids"},
 			},
 
+			"kind": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  string(tfe.Sentinel),
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(tfe.OPA),
+						string(tfe.Sentinel),
+					}, false),
+			},
+
+			"overridable": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
This attribute is not updatable so I think it needs a ForceNew: true prop.

Reply:
The overridable attribute is actually updatable.

Reply:
Ah, OK in that case it should be added to the [update action](https://github.com/hashicorp/terraform-provider-tfe/blob/7d46f92454329f2d1d26c8c7e951ed1907a919b0/tfe/resource_tfe_policy_set.go#L280)

Reply:
https://github.com/hashicorp/go-tfe/pull/594 Added a fix in the client. Unfortunately this will need to be released before I can fix the provider :(